src/drivers/windows/lv_windows_display.c

```diff
@@ -115,6 +119,26 @@ int32_t lv_windows_dpi_to_physical(int32_t logical, int32_t dpi)
  *   STATIC FUNCTIONS
  **********************/
 
+static void __stdcall lv_windows_center_window(HWND hwnd)
```

Comment:
I noticed the Win API calls have its return not checked, as well this function is void, if changing it breaks the API would not be possible to at least put LOG_WARNIG/LOG_ERR to track possible erros that may happen? 

I'm afraid of silent failure here may become hard to debug.

Reply:
I agree with you. Also, I think we should not use stdcall to define LVGL APIs.

Kenji Mouri

Reply:
if stdcall is required in windows platform you can define a noop version when not in Windows version

Reply:
I don't think stdcall is necessary when not calling the Windows APIs.

Reply:
You are right, I have made the changes accordingly.